documentation/jetty/modules/code/examples/src/main/java/org/eclipse/jetty/docs/programming/security/siwe/SignInWithEthereum.java

```diff
@@ -11,27 +11,12 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.security.siwe.example;
+package org.eclipse.jetty.docs.programming.security.siwe;
```

Comment:
We don't need an extra package `siwe`, I think we can just put the documentation examples for security in the `security` package.